### PR TITLE
Feature/v1.1.0

### PR DESCRIPTION
**Pull request by pond**
* Maintenance bundle update brings in Rack 2.x (still conforms to Rack _protocol_ v1.3 in theory)
* AMQP connections now only work (at least on local machine) via explicit `127.0.0.1`, not `localhost`, so change default
* Requires Ruby 2.2.x because of the Rack update, so 2.1.x dropped
* Default to 2.3.1 instead of any earlier version
* Bump from v1.0.0 to v1.1.0 given significant Rack change but no Alchemy Flux API breakage in theory

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.3.1
```
Comment (mbdietrich) [inline .ruby-version]: Is this necessary if the gem isn't standalone?

Comment (pond) [inline .ruby-version]: Probably not, though arguably helpful to local dev if using RBEnv. Can remove?

**Comment by mbdietrich** [comment]
+1

**Comment by daveharris** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -112,4 +112,5 @@ bundle exec ruby examples/example_1_send_message.rb
 * Graham Jenson
 * Tom Cully
 * Andrew Hodgkinson
+* Max Dietrich
```
Comment (mbdietrich) [inline README.md]: :)

**Pull request merged**